wandb/sdk/internal/internal_api.py

```diff
@@ -3513,6 +3522,59 @@ def replace(a: str, b: str) -> None:
         link_artifact: Dict[str, Any] = response["linkArtifact"]
         return link_artifact
 
+    def _resolve_org_entity_name(
+        self, entity: str, organization: str, project: str, portfolio_name: str
+    ) -> str:
+        org_fields = self.server_organization_introspection()
+        can_fetch_org_entity = "orgEntity" in org_fields
+        if not organization and not can_fetch_org_entity:
+            raise ValueError(
+                """Fetching Registry artifacts without inputting an organization is unavailable for your server version.
+                Please upgrade your server to XX or later."""
+            )
+        elif can_fetch_org_entity:
+            org_entity, org_name = self.fetch_org_entity_from_entity(entity)
+            if organization:
+                if organization != org_name and organization != org_entity:
+                    raise ValueError(
+                        f"Input wrong organization: {organization} for registry: {project}/{portfolio_name}"
```

Comment:
```suggestion
                    raise ValueError(
                        f""Artifact belongs to the organization {org_entity} and cannot be linked to {organization}. Please update the target path with the correct organization name.",
```

Should we keep the errors consistent?